tests/broker/kafka/p10-r3-unordered/resources/100-pod.yaml

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
wanna clean out the old `testcases` for "kafka broker", to be replaced by this?

There are some left overs

Reply:
No, these tests are just partially here, as next steps we will clean those up (perf lab still relies on `testcases`)